requirements.txt

```diff
@@ -13,5 +13,5 @@
 #   limitations under the License.
 
 
-grpcio-tools==1.48.1
-mypy-protobuf==3.2
+grpcio-tools==1.54.2
```

Comment:
I think we should use version [1.56.0](https://pypi.org/project/grpcio-tools/1.56.0/) to match the grpc-version in the BOM